### PR TITLE
loader_type_js: add ability to specify error handler #672

### PR DESCRIPTION
**Pull request by hcodes**

### CODE REVIEW

File: common.blocks/loader/_type/loader_type_js.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+modules.define('spec', ['loader_type_js', 'sinon'], function(provide, loader, sinon) {
+
+describe('loader_type_js', function() {
+    it('should call success callback', function(done) {
+        var spyError = sinon.spy();
+
+        loader('data:text/javascript;charset=utf-8,;', function() {
+            spyError.called.should.be.equal(false);
```
Comment (dfilatov) [inline common.blocks/loader/_type/loader_type_js.spec.js]: spyError.should.not.have.been.called;

**Comment by dfilatov** [comment]
lgtm

**Comment by dfilatov** [comment]
And, please, do the same PR to v3.

**Comment by hcodes** [comment]
v3 #693

**Comment by dfilatov** [comment]
Could you rebase your PR?

**Pull request merged**